shell/components/icons.css.js

```diff
@@ -5,7 +5,7 @@ export default
     font-style: normal;
     -webkit-font-feature-settings: "liga";
     -webkit-font-smoothing: antialiased;
-    vertical-align: middle;
+    /*vertical-align: middle;*/
```

Comment:
We don't need? Just double checking.

Reply:
Turns out `vertical-align` can have non-local side-effects (as can `margin`, e.g.) that make it _considered harmful_ in the flex-box era.